pkg/coin/coint_test.go

```diff
@@ -0,0 +1,28 @@
+package coin_test
+
+import (
+	"testing"
+
+	"github.com/zeta-chain/node/pkg/coin"
+)
+
+func TestCoinType_SupportsRefund(t *testing.T) {
+	tests := []struct {
+		name string
+		c    coin.CoinType
+		want bool
+	}{
+		{"ERC20", coin.CoinType_ERC20, true},
+		{"Gas", coin.CoinType_Gas, true},
+		{"Zeta", coin.CoinType_Zeta, true},
+		{"Cmd", coin.CoinType_Cmd, false},
+		{"Unknown", coin.CoinType(100), false},
```

Comment:
Test name full lowercase

Reply:
I think it would be better to keep this as it is , since these are all proper nouns